src/_variables.scss

```diff
@@ -1,2 +1,3 @@
 // Customize theme
 $primary: #203563;
+$bg-color: #FCF5E0;
```

Comment:
Please precise which bg-color it is

Reply:
What do you mean? It's the default bg color. White would be bg-light so I guess it doesn't need more infos